draft-polli-ratelimit-headers.md

```diff
@@ -338,32 +336,22 @@ One example of `RateLimit-Remaining` use is below.
 
 The `RateLimit-Reset` response header field indicates either
 
-- the number of seconds until the quota resets, or
-- the timestamp when the quota resets.
+- the number of seconds until the quota resets.
 
 The header value is:
 
-    RateLimit-Reset = delta-seconds / IMF-fixdate
-
-The `IMF-fixdate` format is defined in [RFC7231] appendix D.
-
-The `RateLimit-Reset` value:
-
-- SHOULD use the `delta-seconds` format;
-- MAY use the `IMF-fixdate` format.
+    RateLimit-Reset = delta-seconds
 
-The `IMF-fixdate` format is NOT RECOMMENDED.
+The delta-seconds format is used because:
 
-The preferred format is the `delta-seconds` one, because:
-
-- it does not rely on clock synchronization and is resilient to clock skew between client and server;
+- it does not rely on clock synchronization and is resilient to clock adjustment
+  and clock skew between client and server (see [RFC7231] Section 4.1.1.1);
 - it mitigates the risk related to thundering herd when too many clients are serviced with the same timestamp.
 
-Two examples of `RateLimit-Reset` use are below.
+An example of `RateLimit-Reset` use is below.
 
 ~~~
-   RateLimit-Reset: 50                              ; preferred delta-seconds notation
-   RateLimit-Reset: Tue, 15 Nov 1994 08:12:31 GMT   ; IMF-fixdate notation
+   RateLimit-Reset: 50
 ~~~
 
 The client MUST NOT give for granted that all its `request-quota` will be restored
```

Comment:
"give for granted" looks like a mangling of the colloquialism "take for granted". Using the word "assume" would be more appropriate for a formal specification.